scipy/special/functions.json

```diff
@@ -1019,11 +1019,6 @@
             "nbdtri": "iid->d"
         }
     },
-    "nbdtrik": {
-        "cdf_wrappers.h": {
-            "cdfnbn2_wrap": "ddd->d"
-        }
-    },
```

Comment:
I removed `nbdtrick` from `functions.json` to replace it from the boost implementation. I don't know how all this works, maybe I would be able to directly tell functions.json to call it from `_boost` dir